src/segmented-control/styles.scss

```diff
@@ -22,13 +27,28 @@
 .segment-part {
   display: -ms-inline-grid;
   display: inline-grid;
+
   @include styles.media-breakpoint-down(styles.$breakpoint-x-small) {
     display: none;
   }
+
+  &.refresh {
+    gap: $segment-space;
+    border-inline: solid 2px awsui.$color-border-input-default;
```

Comment:
should `2px` be `border-width-field`?

Reply:
```suggestion
    border-inline: solid awsui.$border-width-field awsui.$color-border-input-default;
```

Reply:
Yes, absolutely. I forgot to update that